src/components/LeafletMap.vue

```diff
@@ -31,6 +32,16 @@ export default {
       required: true
     },
   },
+  setup() {
```

Comment:
fyi I've currently been using Vue.js with the "options" API. here the setup refers to the "composition" API. I don't know how they both cohabitate ?

Reply:
Ah, my bad, I'll check how to properly do this during the weekend.